### ISSUE TITLE
-+ react-scripts build 

### ISSUE THREAD

**Issue by victorrik** [ckeditor/ckeditor5#6643]
## 📝 Provide detailed reproduction steps (if any)
after create my own build of  "ckeditor5-build-inline", how is showed [here](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/installing-plugins.html#adding-a-plugin-to-a-build) and integrate on my proyect i get this:

npm run build

> pruebas@0.1.0 build C:\Users\NIBIRU_WEB\Documents\htdocs\pruebas
> react-scripts build

Creating an optimized production build...

<--- Last few GCs --->

[6924:000001D9BFD7ED00]    74436 ms: Scavenge 2039.8 (2050.4) -> 2039.2 (2050.7) MB, 10.4 / 0.0 ms  (average mu = 0.324, current mu = 0.290) allocation failure
[6924:000001D9BFD7ED00]    75708 ms: Mark-sweep 2040.1 (2050.7) -> 2038.8 (2050.7) MB, 1270.8 / 0.1 ms  (average mu = 0.231, current mu = 0.027) allocation failure scavenge might not succeed

<--- JS stacktrace --->

==== JS stack trace =========================================

    0: ExitFrame [pc: 00007FF7198D4D3D]
Security context: 0x03058b4408a1 <JSObject>
    1: SourceMapConsumer_allGeneratedPositionsFor [0000009326E32089] [C:\Users\NIBIRU_WEB\Documents\htdocs\pruebas\node_modules\source-map\lib\source-map-consumer.js:~178] [pc=000002D63F698963](this=0x01fcebeac001 <BasicSourceMapConsumer map = 000003508E8B1F31>,0x02ff0e998129 <Object map = 000003508E8AEC41>)
    2: /* anonymous */(aka /* anonymous */) [000...

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory

Writing Node.js report to file: report.20200421.200711.6924.0.001.json
Node.js report completed
 1: 00007FF718D0246F napi_wrap+124559
 2: 00007FF718CA37D6 public: bool __cdecl v8::base::CPU::has_sse(void)const __ptr64+34502
 3: 00007FF718CA4496 public: bool __cdecl v8::base::CPU::has_sse(void)const __ptr64+37766
 4: 00007FF7194A8D9E private: void __cdecl v8::Isolate::ReportExternalAllocationLimitReached(void) __ptr64+94
 5: 00007FF719490DE1 public: class v8::SharedArrayBuffer::Contents __cdecl v8::SharedArrayBuffer::Externalize(void) __ptr64+833
 6: 00007FF71935E6AC public: static void __cdecl v8::internal::Heap::EphemeronKeyWriteBarrierFromCode(unsigned __int64,unsigned __int64,class v8::internal::Isolate * __ptr64)+1436
 7: 00007FF719369A50 public: void __cdecl v8::internal::Heap::ProtectUnprotectedMemoryChunks(void) __ptr64+1312
 8: 00007FF719366584 public: static bool __cdecl v8::internal::Heap::PageFlagsAreConsistent(class v8::internal::HeapObject)+3204
 9: 00007FF71935BE13 public: bool __cdecl v8::internal::Heap::CollectGarbage(enum v8::internal::AllocationSpace,enum v8::internal::GarbageCollectionReason,enum v8::GCCallbackFlags) __ptr64+1283
10: 00007FF71935A5E4 public: void __cdecl v8::internal::Heap::AddRetainedMap(class v8::internal::Handle<class v8::internal::Map>) __ptr64+2356
11: 00007FF71937B8B5 public: class v8::internal::Handle<class v8::internal::HeapObject> __cdecl v8::internal::Factory::NewFillerObject(int,bool,enum v8::internal::AllocationType) __ptr64+53
12: 00007FF7190E7B89 public: class v8::internal::interpreter::JumpTableTargetOffsets::iterator & __ptr64 __cdecl v8::internal::interpreter::JumpTableTargetOffsets::iterator::operator=(class v8::internal::interpreter::JumpTableTargetOffsets::iterator && __ptr64) __ptr64+4057
13: 00007FF7198D4D3D public: virtual bool __cdecl v8::internal::SetupIsolateDelegate::SetupHeap(class v8::internal::Heap * __ptr64) __ptr64+567949
14: 000002D63F698963
npm ERR! code ELIFECYCLE
npm ERR! errno 134
npm ERR! pruebas@0.1.0 build: `react-scripts build`
npm ERR! Exit status 134
npm ERR!
npm ERR! Failed at the pruebas@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\NIBIRU_WEB\AppData\Roaming\npm-cache\_logs\2020-04-22T01_07_12_059Z-debug.log

and this and issue that i get after integrate my custom build but before integrate my custom build i did "npm run build " whit the the build installed with "npm install --save @ckeditor/ckeditor5-react @ckeditor/ckeditor5-build-classic" and all is good, that just occurs with the custom build

**Comment by pomek** [comment]
Could you show us your directory tree? Where do you have the built `ckeditor.js` file?

Webpack/Babel/React scripts… the tools (most probably) ignore stuff inside `node_modules/` (where all libraries are installed). It means, our Inline build is also ignored (not processed by those tools).

However, those tools won't ignore files inside the `src/` directory where (I assume) your application code (including custom build) is placed. It means that the minified ckeditor file (custom build) is being processed by Webpack/Babel and because the minified file is just ugly and hard to parse, Node throws the ugly error (mentioned in the issue).

The issue does not occur for our build because it isn't being processed by Webpack/Babel although our build and your custom build are similar.

Moving the custom build outside `src/` directory or even outside the project directory will help. You can also push your build to a repository on Github and install it as a package (that will land to the `node_modules/` directory).
